src/styles/utils/_variables.scss

```diff
@@ -0,0 +1 @@
+$transition-ease-duration: 0.3s ease;
```

Comment:
Ease is a default value, you can delete it. You can leave only time in this variable.

Reply:
Thanks! I forgot about this...